game/src/ungap/route/mod.rs

```diff
@@ -27,26 +29,45 @@ impl TakeLayers for RoutePlanner {
     }
 }
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
```

Comment:
Here's the strategy for making one world handle objects owned/managed by different chunks of code. The 2 route ID cases are owned by the code here, and `InputWaypoints` is in charge of `ID::Waypoint`